docs/basic-features/data-fetching/swr.md

```diff
@@ -0,0 +1,21 @@
+---
+description: 'Learn how to use SWR, a data fetching React hook that handles caching, revalidation, focus tracking, refetching on interval and more.'
+---
+
+# SWR
```

Comment:
Can we change this to just "client-side" instead of SWR, but still teach SWR? We could also show how to do a `useEffect` data fetch as well.

Reply:
+1 showing a useEffect data fetch example. Would make it a lot more beginner friendly since this is the traditional way most React tutorials and beginners do data fetching.